src/korma/core.clj

```diff
@@ -652,8 +667,17 @@
 
 (defn pk
   "Set the primary key used for an entity. :id by default."
-  [ent pk]
-  (assoc ent :pk (keyword pk)))
+  ([ent pk]
+   (assoc ent :pk (keyword pk)))
+  ([ent pk & more]
+   (assoc ent :pk (cons pk more))))
+
+(defn fk
```

Comment:
new fk helper function to more easily set the :fk key to a key seq; this is optional but handy, see usage in new tests created.